docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md

```diff
@@ -174,34 +174,33 @@ In the web browser, add the **List items** web part to the canvas and open its p
 
 ## Create asynchronous dropdown property pane control
 
-SharePoint Framework offers you a standard dropdown control that allows users to select a specific value. This dropdown control is built in a way that requires all its values to be known upfront. If you want to load the values dynamically or you're loading values asynchronously from an external service and you don't want to block the whole web part, building a custom dropdown control is a viable option.
-
-When using React, a custom property pane control for use with SharePoint Framework web parts consists of a class, that registers the control with the web part, and a React component responsible for rendering the dropdown and managing its data.
+The SharePoint Framework offers you a standard dropdown control that allows users to select a specific value. The dropdown control is built in a way that requires all its values to be known upfront. If you want to load the values dynamically or you're loading values asynchronously from an external service and you don't want to block the whole web part, building a custom dropdown control is a viable option.
 
-> **Note:** In the drop 6 of the SharePoint Framework there is a bug in the Office UI Fabric React Dropdown component that causes the control built in this tutorial to work incorrectly. A temporary workaround is to edit the **node_modules/@microsoft/office-ui-fabric-react-bundle/dist/office-ui-fabric-react.bundle.js** file and change line **12027** from:
-
-```js
-isDisabled: this.props.isDisabled !== undefined ? this.props.isDisabled : this.props.disabled
-```
+When creating a custom property pane control that uses React in the SharePoint Framework, the web part consists of a class that registers the control with the web part, and a React component that renders the dropdown and manages its data.
```

Comment:
This is not quite correct. It's the custom property pane control not the web part that consists of a class that registers the control with the web part and a component for rendering the control.